applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -671,7 +671,7 @@ int le_audio_play_pause(void)
 	return ble_mcs_play_pause(default_conn);
 }
 
-int le_audio_send(uint8_t const *const data, size_t size)
+int le_audio_send(struct encoded_audio enc_audio)
```

Comment:
Should we check enc_audio.num_ch here, and e.g. throw a warning if it is more than one?

Reply:
Changed